src/event_object_movement.c

```diff
@@ -2341,12 +2341,10 @@ u8 CreateCopySpriteAt(struct Sprite *sprite, s16 x, s16 y, u8 subpriority)
 
 void SetObjectEventDirection(struct ObjectEvent *objectEvent, u8 direction)
 {
-    s8 d2;
     objectEvent->previousMovementDirection = objectEvent->facingDirection;
     if (!objectEvent->facingDirectionLocked)
     {
```

Comment:
The curly brackets here can be removed. The project doesn't lay out a scope for `if` statements that execute 1 single operation.